mk/common.mk

```diff
@@ -10,6 +10,8 @@ ifeq ($(UNAME_M),x86_64)
     HOST_PLATFORM := x86
 else ifeq ($(UNAME_M),aarch64)
     HOST_PLATFORM := aarch64
+else ifeq ($(UNAME_M),arm64) # macOS
+    HOST_PLATFORM := aarch64
```

Comment:
@vacantron, I am not sure for this since it is related to the manipulation in `mk/artifact.mk`. Can you check?

Reply:
We only run the architecture test on the x86-64 and the Aarch64 because of the lack of the Sail model . I think it is better to use `Arm64` to distinguish with `Aarch64` targets for fetching the model in the future.